common/src/main/java/com/sequenceiq/cloudbreak/telemetry/DataBusEndpointProvider.java

```diff
@@ -11,6 +14,11 @@
 
     private static final String CNAME_ENDPOINT = "https://dbusapi.us-west-1.altus.cloudera.com";
```

Comment:
I think this should be a config property

Reply:
resolving this as CB-11658 will cover that